README.md

```diff
@@ -27,20 +27,20 @@ Each job can be configured with one Gerrit server.
 
 
 # Environments
-* `linux`
-    * `java-1.6`
-        * `maven-3.0.4`
-* Java 1.7: Not officially supported, but if you prefer you can use it
-  for development. Patches to fix problems with Java 1.7 are welcome.
+* Supported deployment runtime: minimum Java 1.6
```

Comment:
No.

I didn't realize that README.md was both a readme and also contained build
instructions.

I've reverted the modifications to the build instructions.


On Wed, May 20, 2015 at 3:11 PM, Robert Sandell <notifications@github.com>
wrote:

> In README.md
> <https://github.com/jenkinsci/gerrit-trigger-plugin/pull/235#discussion_r30699077>
> :
>
> > @@ -27,20 +27,20 @@ Each job can be configured with one Gerrit server.
> >
> >
> >  # Environments
> > -* `linux`
> > -    * `java-1.6`
> > -        * `maven-3.0.4`
> > -* Java 1.7: Not officially supported, but if you prefer you can use it
> > -  for development. Patches to fix problems with Java 1.7 are welcome.
> > +* Supported deployment runtime: minimum Java 1.6
>
> I'm not sure this follows the literate spec
> <https://github.com/jenkinsci/literate-plugin> It might work. Have you
> tested it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/gerrit-trigger-plugin/pull/235/files#r30699077>
> .
>



-- 
Øyvind Harboe - Can Zylin Consulting help on your project?
http://www.zylin.com/